container/cli.py

```diff
@@ -107,6 +107,8 @@ def subcmd_build_parser(parser, subparser):
     subparser.add_argument('--services', action='store',
                            help=u'Rather than perform an orchestrated build, only build specific services.',
                            nargs='+', dest='service', default=None)
+    subparser.add_argument('--use-vault-password-file', action='store_true',
```

Comment:
This looks like it would have a different behavior from the ansible `vault-password-file` where users can specify the path (to somewhere outside the repo, for example). Do you think the true/false option here covers all the use cases for vault passwords?